.travis.yml

```diff
@@ -24,6 +24,8 @@ script:
     //src/...
     //src_kt/...
   - npm run test-with-start
+  - npm run build:rollup
+  - npm --prefix=server test
 
```

Comment:
I hope we can eventually shard this out to be tested separately, since it doesn't (?) impact the rest of the codebase.

Reply:
server tests have been disabled for the time being.